src/components/Header/Header.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "./Header.scss";
+
+function Header() {
+  return (
+    <div className="header">
+      <div className="header-title">To-do list</div>
+      <div className="header-logo">
```

Comment:
Acredito que essa div é desnecessária. Poderíamos ter somente a `img` e no sass agente poderia trabalhar com a `img` ao invés de colocar dentro de uma classe fincando algo parecido com:

```
// Header.scss

.header {
  ...
  img {
     ...
  }
}
```

Reply:
I agree with you, already made the changes!